src/api/app/assets/stylesheets/webui2/colors.scss

```diff
@@ -4,6 +4,8 @@ $gray-150: #f0f0f0;
 $gray-200: #e9ecef;
 $gray-400: #aeaeae;
 $gray-550: #878c92;
+$white: #ffffff;
+$black: #000000;
```

Comment:
This seems super generic. Too generic.

Reply:
:thinking:  What do you propose? using a different variable name or not using variables for that kind of "generic" colors? (I created a variable for consistency).